src/client/cli/cmd/stop.cpp

```diff
@@ -84,7 +84,11 @@ mp::ParseCode cmd::Stop::parse_args(mp::ArgParser* parser)
     QCommandLineOption time_option({"t", "time"}, "Time from now, in minutes, to delay shutdown of the instance",
                                    "time", "0");
     QCommandLineOption cancel_option({"c", "cancel"}, "Cancel a pending delayed shutdown");
-    parser->addOptions({all_option, time_option, cancel_option});
+    QCommandLineOption force_option(
+        "force",
+        "Immediately force switching the instance off. This could potentially corrupt a running"
```

Comment:
Our [documentation](https://multipass.run/docs/use-an-instance#heading--Stop-an-instance-forcefully) says more about this. 
Force stop is used when the state is non-responsive, unknown or suspended. Do we just want to mentioned suspended state here?

Reply:
BTw, now the new help text is `Force the instance to shut down immediately. Warning: This could potentially corrupt a running instance, so use with caution.` after the iteration with technical author.

Reply:
I would be fine with replacing that version in.